docs/stake.md

```diff
@@ -225,6 +240,7 @@ For more details around all the commands of `razor-go`, please check out the `or
 If you would rather install from source, please follow Instructions here to [run a Razor Network node from source](https://github.com/razor-network/oracle-node#building-the-source).
 
 ## Update docker image
+> **_NOTE:_**: _If you are updating to `v2` in the same machine where `v1` node was running, make sure you delete all the files in ` .razor/data_files` directory.
```

Comment:
If you are updating to v2 on the same machine where the v1 node was running, make sure to delete all the files in the .razor/data_files directory.

Reply:
Done in https://github.com/rajkharvar/docs-v2/pull/2/commits/7e9762ff5494e536401b6ea1517f441014fdfd30